### DISCUSSION TITLE
로그인, 회원가입 에러

### DISCUSSION THREAD

**Discussion by june082** [code-bootcamp/odisca_client#38] [제보하기]
refreshtoken을 생성하는 과정에서 기존에는 없었던 로그인, 회원가입 에러가 발생했던 것 같습니다. 

5월 16일 refreshtoken 작업목록입니다.

1. Src/components/commons/apollo/index.tsx 변경
2. Src/commons/stores/index.ts 변경
3. Src/commons/libraries 안에 getAccessToken.ts 생성
4. Graphql-request 설치
5. Src/components/units/user/login/userLogin.index.tsx 변경

**Comment by tjdgoz** [comment]
cors에러 백엔드 확인중입니다!!
